actioncable/lib/action_cable/test_helper.rb

```diff
@@ -0,0 +1,68 @@
+module ActionCable
+  # Provides helper methods for testing Action Cable broadcasting
+  module TestHelper
+    extend ActiveSupport::Concern
+
+    included do
+      def before_setup # :nodoc:
+        server = ActionCable.server
+        test_adapter = ActionCable::SubscriptionAdapter::Test.new(server)
+
+        @old_cable_adapter = server.pubsub
+
+        server.instance_variable_set(:@pubsub, test_adapter)
+        super
+      end
+
+      def after_teardown # :nodoc:
+        super
+        ActionCable.server.instance_variable_set(:@pubsub, @old_cable_adapter)
+      end
+
+      def assert_transmissions(number, channel)
+        if block_given?
+          original_count = transmissions_size(channel)
+          yield
+          new_count = transmissions_size(channel)
+          assert_equal number, new_count - original_count, "#{number} transmissions to #{channel} expected, but #{new_count - original_count} were sent"
+        else
+          actual_count = transmissions_size(channel)
+          assert_equal number, actual_count, "#{number} transmissions to #{channel} expected, but #{actual_count} were sent"
+        end
+      end
+
+      def assert_no_transmissions(channel, &block)
+        assert_transmissions 0, channel, &block
+      end
+
+      def assert_transmited_data(data, channel)
+        serialized_msg = ActiveSupport::JSON.encode(data)
+        new_messages = transmissions(channel)
+        if block_given?
+          old_messages = new_messages
+          clear_messages(channel)
+          yield
+          new_messages = transmissions(channel)
+          clear_messages(channel)
+
+          # Restore all sent messages
+          (old_messages + new_messages).each { |m| cable_adapter.broadcast(channel, m) }
+        end
+        assert_includes new_messages, serialized_msg, "No messages sent with #{data} to #{channel}"
+      end
+
+      def cable_adapter
```

Comment:
Can we call this `subscription_adapter` like we do in other parts of ActionCable?